src/lib/libraries/extensions/index.jsx

```diff
@@ -235,8 +236,8 @@ const menuItems = [
         iconURL: 'https://editor.snail-ide.com/Screensharing.png', // please forgive me the text is slightly offcenter
         collaborator: 'pooiod7',
         tags: ['penguinmod'],
-        description: 'Share your screen and get the current frame as a image.',
-        featured: true
+        description: 'Share your screen and get the current frame as an image. (replaced by VideoSharing)',
+        featured: false // This is actually an unfinished version of "VideoSharing" (kept to prevent old projects from braking)
```

Comment:
Setting featured to true made the Screensharing item small, instead of hiding it. I don't think removing the item from the library without removing the extension will break projects, as the library is just a way to select extensions.